test/intl402/DateTimeFormat/prototype/format/numbering-system.js

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-formatdatetimepattern
+description: |
+  Checks that numberingSystem property used correctly.
+info: >
+  11.5.5 FormatDateTimePattern ( dateTimeFormat, format, pattern, epochNanoseconds )
+    ...
+    3. Perform ! CreateDataPropertyOrThrow(nfOptions, "numberingSystem", dateTimeFormat.[[NumberingSystem]]).
+    ...
+    8. Perform ! CreateDataPropertyOrThrow(nf2Options, "numberingSystem", dateTimeFormat.[[NumberingSystem]]).
+    ...
+    11. If format has a field [[fractionalSecondDigits]], then
+    ...
+      d. Perform ! CreateDataPropertyOrThrow(nf3Options, "numberingSystem", dateTimeFormat.[[NumberingSystem]]).
+
+locale: [en-US]
```

Comment:
I'm not sure about this, but I think you need to include `en-US-u-nu-arab` and the others in this list as well.